.changeset/metal-stingrays-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-button": patch
```

Comment:
Should this be a `major` version change because we removed the `khanmigo` theme from `Button`? Or is it a patch because we've already migrated off of the khanmigo theme usage from before? Thought I'd check!

Reply:
The latter, I used `patch` b/c I'm removing the khanmigo button in the webapp PR https://github.com/Khan/webapp/pull/32895. I intended to make that `webapp` change first, that way we don't have to deal with breaking changes :). Hope that makes sense.